pages/watering.js

```diff
@@ -55,8 +53,8 @@ export default function Watering() {
       <BackArrow link="/" />
       <Image src="/img/iconWater.png" width={80} height={80} alt="iconWater" />
       <p>Day of the week : {dayOfWeek}</p>
-      {/* only for simulation */}
-      <button onClick={nextDay}>Next day</button> {/* only for simulation */}
+
+      <button onClick={nextDay}>Next day</button>
```

Comment:
When working with a button element, it never hurts to be specific about its type, like type="button" in this case